### ISSUE TITLE
DynamoDB correction

### ISSUE THREAD

**Issue by dimaqq** [scylladb/scylla#10015]
Sorry if this is the wrong place to post this 🙈 

Your website has an article comparing Cassandra and DynamoDB models at https://www.scylladb.com/resources/cql-vs-dynamodb-api/ which includes the following statement about DynamoDB:

> All writes are isolated from each other; two concurrent writes to the same row will be serialized.

This is [probably] true in the default case, but it is decidedly false when the DynamoDB table is a **global table**.

Specifically, if one:
* start with an item `{[hash, range], x: 1, y: 1}`
* send a conditional update `set x = 2` (whatever condition) to region 1
* send a conditional update `set y = 3` (whatever condition) to region 2
* at about the same time
* both updates succeed

The result may be:
* `{[hash, range], x: 2, y: 3}` or
* `{[hash, range], x: 1, y: 3}` or
* `{[hash, range], x: 2. y: 1}`

I believe it's documented somewhere, although not prominently.

**Comment by nyh** [comment]
You are right - this statement (copied from my blog post https://www.scylladb.com/2020/05/12/comparing-cql-and-the-dynamodb-api/) is about writes to normal, non-Global, DynamoDB tables.

I'm not sure which change would make sense to the text - perhaps just adding the word "local" (or "non-global") somewhere in that paragraph to emphasize that it is about non-global tables?

If we want to go forward with this issue, we should move it to the scylla docs bug tracker - this text is not in scylla.git.

**Comment by dimaqq** [comment]
Maybe `except “global tables” which offer weaker guarantee`, I think that may be called LWW 🤔

**Comment by swasik** [comment]
@nyh who has the permission to modify the blog post? It is adding few words - let us do this and have one less issue.

**Comment by nyh** [comment]
> [@nyh](https://github.com/nyh) who has the permission to modify the blog post? It is adding few words - let us do this and have one less issue.

@swasik note that this is NOT a blog post (the blog post is https://www.scylladb.com/2020/05/12/comparing-cql-and-the-dynamodb-api/). I don't know where the content of https://www.scylladb.com/resources/cql-vs-dynamodb-api/ comes from. It doesn't seem to come from scylla.git. Probably @tzach can answer this.
